.travis.yml

```diff
@@ -11,8 +11,18 @@ addons:
       - doxygen
       - sshpass
       - ant
+      - git
```

Comment:
Both git and cmake should already be included in the image, no?

Reply:
CMake is only installed for the current user, so when we run as root it falls over. Installing system-wide avoids us having to mess with paths

Adding git too was a result of mind block though